Makefile

```diff
@@ -125,6 +125,9 @@ TFTP_IPRANGE ?= 192.168.100
 export TFTP_IPRANGE
 TFTPD_DIR ?= build/tftpd/
 
+# Default TFTP Server Port to IANA well known UDP/69
+TFTP_SERVER_PORT ?= 6069
+
```

Comment:
Doesn't this need to be exported?

Reply:
The code previously assumed that the user had exported `TFTP_SERVER_PORT` themselves if they wanted a non-root port; based on your change in pull request and discussion it is now exported both in `Makefile` and also in `make env` target that eg, `scripts/build-qemu.sh` uses. This appears to be sufficient that everything sees the default port as UDP/6069, but also allow overriding to use UDP/69 if desired (`TFTP_SERVER_PORT=69`).

Beware that this means that users who built their BIOS a while ago will be trying to TFTP from UDP/69, but their TFTP server will no default to starting on UDP/6069, which may cause additional confusion.... (defaulting to TFTP server on UDP/69 was more backwards compatible).